typescript/sdk/src/providers/transactions/schemas.ts

```diff
@@ -1,7 +1,18 @@
 import { z } from 'zod';
 
+import { ZHash } from '../../metadata/customZodTypes.js';
+
+export const BigNumberSchema = z.string();
+
 export const PopulatedTransactionSchema = z.object({
   to: z.string(),
   data: z.string(),
+  value: BigNumberSchema.optional(),
   chainId: z.number(),
 });
+
+export const CallDataSchema = z.object({
+  to: ZHash,
+  data: z.string(),
+  value: BigNumberSchema.optional(),
+});
```

Comment:
why have 2 different schemas which share 3 of same fields?
also `to` is `ZHash` on `CallData` but `z.string` on `PopulatedTransaction`

Reply:
> why have 2 different schemas which share 3 of same fields?

to emulate the current state of the repository. we can eventually refactor everything to just use a single in-house type. captured here: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3845

> also to is ZHash on CallData but z.string on PopulatedTransaction

did not see this– updated PopulatedTransaction to use ZHash